Tools/debug/CollectSystemInfo.ps1

```diff
@@ -88,6 +88,75 @@ function Export-EventLogs {
     }
 }
 
+function Ensure-DiskSpd {
+    $scriptDir = $null
+    try {
+        $scriptDir = Split-Path -Parent $MyInvocation.MyCommand.Definition
+    } catch {}
+    if (-not $scriptDir -or $scriptDir -eq '' -or -not (Test-Path $scriptDir)) {
+        $scriptDir = (Get-Location).Path
+    }
+
+    $diskspdExe = Join-Path $scriptDir 'diskspd.exe'
+    $zipPath = Join-Path $scriptDir 'DiskSpd.ZIP'
+    $extractDir = Join-Path $scriptDir 'DiskSpdExtracted'
+
+    if (-not (Test-Path $diskspdExe)) {
+        if (-not (Test-Path $zipPath)) {
+            Write-Warning "DiskSpd.zip not found. Please place it next to the script."
```

Comment:
1. Currently, we need to provide 2 files for collecting logs. One of them is binary, so better to check the file checksum to make sure that it is not replaced with some `virus.zip`
2. Update the README with information that DiskSpd.zip is required to be copied too.


@YanVugenfirer, what about downloading DiskSpd.zip from GitHub if it is not found, but an Internet connection is present?

Reply:
diskspd uses DeviceIOControl to get IO information, it will be complicated to rewrite on powershell